### PR TITLE
update script for source_protocol_ver

### PR DESCRIPTION
**Pull request by ljmen**
reduce the loop times when setting up nfs server

Singed-off-by: ljmen <lmen@redhat.com>

### CODE REVIEW

**Comment by ljmen** [comment]
@lento-sun 
updated the script,pls have a check,thanks

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -442,6 +442,12 @@ def setup_or_cleanup_nfs(is_setup, mount_dir="nfs-mount", is_mount=False,
                               "\"setsebool virt_use_nfs on\" to get "
                               "nfs access right.")
         _nfs.setup()
+        nfs_mount_info = process.run('nfsstat -m', shell=True).stdout.decode().strip().split(",")
```
Comment (dzhengfy) [inline virttest/utils_test/libvirt.py:445]: 'stdout.decode()'  should be replaced by 'stdout_text' for better compatibility.

**Comment by dzhengfy** [comment]
And the test results for some representative test cases are required to be provided here.

**Comment by ljmen** [comment]
test result pass:

[root@localhost tp-libvirt]#  avocado run  --vt-type libvirt --vt-machine-type q35 type_specific.io-github-autotest-libvirt.virsh.pool.positive_test.pool_type_netfs.source_format_nfs
/opt/code/avocado/avocado/plugins/run.py:286: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast and keep-tmp. 
  FutureWarning)
JOB ID     : 6f8914223b0ece66868f9ce10eb9734db601765a
JOB LOG    : /root/avocado/job-results/job-2020-09-17T22.36-6f89142/job.log
 (1/2) type_specific.io-github-autotest-libvirt.virsh.pool.positive_test.pool_type_netfs.source_format_nfs.with_source_protocol_ver: PASS (7.15 s)
 (2/2) type_specific.io-github-autotest-libvirt.virsh.pool.positive_test.pool_type_netfs.source_format_nfs.no_source_protocol_ver: PASS (6.83 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 16.04 s

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**